src/types.jl

```diff
@@ -566,6 +566,116 @@ function System(;
                     loggers, k_converted, force_units, energy_units, atom_masses)
 end
 
+"""
+    System(crystal; <keyword arguments>)
+
+A physical system to be simulated, constructed from a SimpleCrystals.jl `Crystal` struct.
+Properties unused in the simulation or in analysis can be left with their
+default values.
+`atoms`, `atoms_data`, `coords` and `boundary` are automatically calcualted from the cyrstal struct. Extra
+atom paramaters like `σ` will have to be added manually after construction using the convenience constructor:
+System(system; <keyword arguments>)
+
+
+This is a sub-type of `AbstractSystem` from AtomsBase.jl and implements the
```

Comment:
No need for this or the below arguments line, this docstring gets appended to the previous.